lib/Utils/Logging.pm

```diff
@@ -78,13 +78,13 @@ sub tar_and_upload_log {
 
  save_and_upload_systemd_unit_log($unit);
 
-Saves the journal of the systemd unit C<$unit> to C<journal_$unit.log> and uploads it to openQA.
+Saves the journal of the systemd unit C<$unit> to C<journal_$unit.log.txt> and uploads it to openQA.
 
 =cut
 
 sub save_and_upload_systemd_unit_log {
     my ($self, $unit) = @_;
-    $self->save_and_upload_log("journalctl --no-pager -u $unit -o short-precise", "journal_$unit.log");
+    $self->save_and_upload_log("journalctl --no-pager -u $unit -o short-precise", "journal_$unit.log.txt");
```

Comment:
```suggestion
    $self->save_and_upload_log("journalctl --no-pager -u $unit -o short-precise", "journal_$unit.txt");
```

`.log` is artificial from the beginning . no point to make it `.log.txt` just use `.txt` if it is what you want